microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -39,11 +39,31 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
+
+To follow the least privilege model for the applications in your Azure AD tenant, you might be concerned about consenting to many permissions scopes to the Microsoft Graph PowerShell app over time. To avoid this, you can register your own app for use with Microsoft Graph PowerShell. This allows you to have more granular control.
```

Comment:
I would rephrase this:
To follow the least privilege model for the applications in your Azure AD tenant, you might be concerned about consenting to many permissions scopes to the Microsoft Graph PowerShell app over time. To avoid this, you can register your own app for use with Microsoft Graph PowerShell. This allows you to have more granular control.

Follow the steps below to set up your own app:

To:
Follow the steps below to create custom applications that can be used to connect to Microsoft Graph PowerShell. Use this approach if you need to limit the consent permissions granted for Graph PowerShell usage.